be/src/runtime/mem_tracker.h

```diff
@@ -588,14 +579,18 @@ class PoolMemTrackerRegistry {
   /// with the process tracker as its parent. There is no explicit per-pool byte_limit
   /// set at any particular impalad, so newly created trackers will always have a limit
   /// of -1.
+  /// TODO(cmy): this method is not used for now. the memtracker returned from here is
+  ///            got from a shared_ptr in `pool_to_mem_trackers_`.
+  ///            I don't know why it need to return a raw pointer.
+  ///            If this method be further used, please notice that.
```

Comment:
GetRequestPoolMemTracker is from
https://github.com/cloudera/Impala/blob/495397101e5807c701df71ea288f4815d69c2c8a/be/src/runtime/mem-tracker.h#L497
So we didn't make it shared last time, just Ieave it as it is. Maybe we could return shared ptr, I think there should be no problem.

Reply:
Okay, since this function has not been used yet, I will change the return value to shared_ptr